docs: add rancher ui installation instructions

salasberryfin opened the pull request:
## Description

This PR adds instructions for installing the Turtles Helm chart via Rancher UI (adding the repository and then installing the chart via the local cluster). This is the same installation process we use in the demos and it's a simpler alternative to installing it manually via `helm install`.

With this change, the installation options are grouped into two different categories:
- Via UI
- Via Helm install

The UI installation is now the recommended approach and links from other pages in the docs point to this option. It also includes a GIF demo to visually show how the extension is installed.

## What does it look like?

![image](https://github.com/rancher/turtles-docs/assets/40443040/b63b5ad9-8b6c-4ece-b08d-d5c3f3e85300)
![image](https://github.com/rancher/turtles-docs/assets/40443040/f2fe94d2-59ed-47f1-945d-973c7647344f)
![image](https://github.com/rancher/turtles-docs/assets/40443040/9d2cf0b6-2205-400b-8b5c-9808a0b9aa47)

Fixes: #73

furkatgofurov7 commented:
Mostly looks good, can we remove "Operator" word from the uninstall page as well to be consistent with install?

salasberryfin commented:
I updated the changes and modified the uninstall section title + added a warning related to not using the UI when installing Cluster API Operator separately.

richardcase commented on docs/getting-started/intro.md:
```diff
@@ -14,7 +14,7 @@ Everything you need to know about getting started with Rancher Turtles.
-| Cert-manager             | `>=v1.12.0`                              | Using [helm](https://cert-manager.io/docs/installation/helm/#installing-with-helm) based installation or via [kubectl apply](https://cert-manager.io/docs/installation/#default-static-install) command or as a helm [dependency](./install_turtles_operator.md) (recommended)                                                          ...
-| Cluster API Operator     | `>=v0.5.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) install or as a [dependency](./install_turtles_operator.md#install-cluster-api-operator-as-a-helm-dependency) (recommended) to the `Rancher Turtles`                                                      ...
+| Cert-manager             | `>=v1.12.0`                              | Using [helm](https://cert-manager.io/docs/installation/helm/#installing-with-helm) based installation or via [kubectl apply](https://cert-manager.io/docs/installation/#default-static-install) command or as a helm [dependency](./install-rancher-turtles/using_rancher_dashboard.md) (recommended)                                   ...
+| Cluster API Operator     | `>=v0.5.1`                               | Using [helm chart](https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/README.md#method-2-use-helm-charts) and then [manually installing Turtles](./install-rancher-turtles/using_helm.md) or letting Turtles take care of it as a [dependency](./install-rancher-turtles/using_rancher_dashboard.md) (recommended)  ...
```
I'd probably switch these so the recommended option is first (i.e. via the turtles chart) and the helm method second (and maybe marked as advanced use cases)

furkatgofurov7 replied:
@richardcase did you mean: https://github.com/rancher/turtles-docs/pull/75/commits/e5feb2ae971d0bb014226a44f4ecf31b74c2490f?

richardcase reviewed (commented):
Looks good to me. Just one comment around switching the order.

furkatgofurov7 commented:
I will address any open comments on this one since @salasberryfin will be on PTO next week.

furkatgofurov7 commented:
I also [removed](https://github.com/rancher/turtles-docs/pull/75/commits/67b5ead86d8bf4220227783795d62261cb1734d3) all "operator" word from Rancher Turtles mentions in the latest docs (excluding the ones in versioned_docs/* folder) to align on what we agreed in https://github.com/rancher/turtles-docs/pull/75#issuecomment-2022577430

richardcase commented on docs/getting-started/install-rancher-turtles/using_rancher_dashboard.md:
```diff
@@ -0,0 +1,52 @@
+
+### Installation
+
+- From your browser, access Rancher Manager and navigate to the local cluster.
```
```suggestion
- From your browser, access Rancher Manager and explore the **local** cluster.
```

furkatgofurov7 replied:
Thanks, done

richardcase commented on docs/getting-started/uninstall_turtles.md:
```diff
@@ -14,11 +14,11 @@ To simplify uninstalling Rancher Turtles (via Rancher Manager or helm command),
 :::
 
-There are two options to uninstall the Rancher Turtles Operator depending on the installation method.
+There are two options to uninstall the Rancher Turtles depending on the installation method.
```
As discussed we will only mention uninstalling via helm on the cli initially. But this can be changed in #77. wdyt @furkatgofurov7 @Danil-Grigorev?

furkatgofurov7 replied:
Let's change it in #77 👍🏼

richardcase reviewed (commented):
Looks good to me. Just a couple of minor comments.
